backend/backend.go

```diff
@@ -642,6 +642,17 @@ func (backend *Backend) AccountsTotalBalanceByKeystore() (map[string]KeystoreTot
 			)
 			currentTotal.Add(currentTotal, fiatValue)
 		}
+		// Add lightning balance if it is enabled.
+		if len(backend.Config().LightningConfig().Accounts) > 0 {
```

Comment:
Could you please add a `lightningConfig.LightningEnabled()` method that returns `len(backend.Config().LightningConfig().Accounts) > 0` inside config/lightning.go, and use it to check if lightning is enabled? this way we avoid duplicating the check around the code. We may want to update it in the future.

Reply:
that will be better, `LightningEnabled` added